tests/test_config_simple.py

```
@@ -20,7 +20,10 @@ def test_copy_to_clipboard_termux_success(mocker):
 
     assert result is True
     mock_subprocess.assert_called_once_with(
-        ["termux-clipboard-set"], input="test text".encode("utf-8"), check=True
+        ["termux-clipboard-set"],
+        input="test text".encode("utf-8"),
```

Ruff (0.15.12)

[warning] Unnecessary call to `encode` as UTF-8

Rewrite as bytes literal

(UP012)